demoapp/composeApp/src/commonMain/kotlin/at/asitplus/cryptotest/App.kt

```diff
@@ -300,18 +310,20 @@ internal fun App() {
                                 when (val alg = keyAlgorithm.algorithm) {
                                     is SignatureAlgorithm.ECDSA -> {
                                         this@createSigningKey.ec {
-                                            curve = alg.requiredCurve ?:
-                                                ECCurve.entries.find { it.nativeDigest == alg.digest }!!
+                                            curve = alg.requiredCurve
+                                                ?: ECCurve.entries.find { it.nativeDigest == alg.digest }!!
```

Comment:
also willing to be convinced, but to me having the elvis operator in the first line makes it clearer that "this statement is not finished yet" when reading

Reply:
pretty sure this is your code, just run through the auto-formatter

Reply:
yes, and the thing i am objecting to is you moving the elvis operator off of the first line :)